packages/amplify-codegen-e2e-tests/test-apps/ts/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "ts",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.7.13",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "aws-amplify": "^4.3.43",
```

Comment:
TODO reminder: we need to bump the version if there is a major version bump for `aws-amplify`

Reply:
This should already be v5 actually. Good catch

Reply:
Changed to `latest` so that it is always up to date.